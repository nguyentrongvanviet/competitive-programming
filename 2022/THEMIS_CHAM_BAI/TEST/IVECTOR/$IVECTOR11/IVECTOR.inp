52
45 51 43 23 12 13 41 2 52 7 47 29 19 39 32 28 48 5 22 9 31 42 10 33 21 20 6 18 40 25 27 24 14 35 50 46 11 38 8 34 3 26 30 37 49 36 16 1 4 15 17 44 
38 24 43 17 17 1 11 40 3 42 31 34 31 6 8 2 16 5 1 3 9 12 1 14 16 21 18 5 5 1 14 2 13 6 1 12 13 13 4 9 10 6 3 7 4 2 2 0 3 2 0 0 