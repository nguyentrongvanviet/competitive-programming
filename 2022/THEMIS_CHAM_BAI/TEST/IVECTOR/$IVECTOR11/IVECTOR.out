47 7 39 45 16 24 8 34 16 18 30 4 4 24 35 34 34 19 8 17 16 12 3 17 15 21 15 9 6 18 9 7 9 14 10 15 13 12 6 8 3 6 2 8 0 5 2 2 3 2 0 0 
48 6 19 16 9 23 30 20 14 18 35 15 7 32 51 28 29 4 5 46 21 43 39 34 2 17 22 47 45 42 24 52 50 25 40 31 27 11 1 13 33 44 12 26 36 3 41 8 37 38 49 10 